src/Service/Program.cs

```diff
@@ -119,6 +121,18 @@ public static ILoggerFactory GetLoggerFactoryForLogLevel(LogLevel logLevel)
                     builder.AddFilter(category: "Microsoft", logLevel);
                     builder.AddFilter(category: "Azure", logLevel);
                     builder.AddFilter(category: "Default", logLevel);
+
+                    // For Sending all the ILogger logs to Application Insights
+                    if (Startup.AppInsightsOptions.Enabled && appTelemetryClient is not null)
+                    {
+                        builder.AddApplicationInsights(configureTelemetryConfiguration: (config) =>
+                            config.ConnectionString = Startup.AppInsightsOptions.ConnectionString,
+                            configureApplicationInsightsLoggerOptions: (options) => { }
+                        )
+                        .AddFilter<ApplicationInsightsLoggerProvider>("", logLevel)
+                        .AddProvider(new ApplicationInsightsLoggerProvider(appTelemetryClient));
```

Comment:
Won't need this (see comments in app insights logger provider file), because `.AddApplicationInsights()` already does this.